index.html

```diff
@@ -175,71 +184,177 @@
 
 ---
 
-## Migrating to Git
+class: middle, center
+
+## When can you start migrating to Git? 
 
-### When can you start? 
 --
-**.okay[Right now!]**
+
+### .okay[Right now!]
+---
+
+## Migrating to Git
+
+### How can you do it?
+
 --
 
-- TFS supports Git and mixing Git/TFVC repos in the same project.
-- Git is the default version control provider for new projects in Azure DevOps (Online & On-Prem).
-- Microsoft advises using Git over TFVC.
+#### 1. Add a Git repository to your TFS Team Project.
+
+- TFS supports mixing Git and TFVC repos in the same project.
 - Integrated Git support in Visual Studio and Visual Studio Code.
   - You will want to learn some basic commands.
 
-- [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+???
+- Git is the default version control provider in TFS and Azure DevOps.
+
+--
+
+#### 2. "Clone" your TFVC into a local Git repository.
+
+- There's a tool for that! .okay[**git-tfs**]
+  - Recommended by [Microsoft](https://docs.microsoft.com/en-us/azure/devops/learn/git/migrate-from-tfvc-to-git#advanced-migrations), [GitLab](https://docs.gitlab.com/ee/user/project/import/tfvc.html) and Git-SCM's [Pro Git Book](https://git-scm.com/book/en/v2/Git-and-Other-Systems-Migrating-to-Git#_git_tfs).
+  - Tool takes a long time to process. (~30 mins for 1 GB TFVC repo)
+
+--
+
+#### 3. Push local Git repo to Git hosting service.
+
+- One basic command: `git push`
 ---
 
 ## Migration Tool
 
-### git-tfs
+### git-tfs - Sample Run
 
-- Recommended by Microsoft and Pro Git Book on [Git-SCM](https://git-scm.com/book/en/v2/Git-and-Other-Systems-Migrating-to-Git#_git_tfs).
+```cs
```

Comment:
```suggestion
```bash
```

Reply:
The rendered highlight didn't work really well in either `bash` (.sh scripts) or `shell` (terminal commands). I arbitrarily chose `cs` because it rendered comments in a color that wasn't light grey (more contrast with background).

Reply:
Right, ok. My other comment was in relation to the "comment" that are also a list, so I'll resolve this.